.rubocop.yml

```diff
@@ -0,0 +1,11 @@
+AllCops:
+  NewCops: enable
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Style/Documentation:
+  Enabled: false
+
+Metrics/LineLength:
+  Max: 120
```

Comment:
Please, add a new line at the end of each file.

Reply:
fixed